tracing_android_trace/README.md

```diff
@@ -29,7 +29,7 @@ Note that this does not currently support `tracing` *events*, only spans.
 This limitation is due to the underlying Android platform APIs.
 
 <figure>
-<img src="https://github.com/DJMcNab/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
+<img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
```

Comment:
```suggestion
<img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread.">
```

Reply:
Looks like this and the following one were marked as resolved but no corresponding commit was made.

Reply:
Oops. Not sure how I managed that

Thanks.